src/wazuh_db/wdb.c

```diff
@@ -1049,6 +1049,43 @@ cJSON* wdb_exec_stmt_sized(sqlite3_stmt * stmt, const size_t max_size, int* stat
     return result;
 }
 
+int wdb_exec_stmt_send(sqlite3_stmt* stmt, int peer) {
+    int status = SQLITE_ERROR;
+    if (!stmt) {
+        mdebug1("Invalid SQL statement.");
+        return status;
+    }
+    cJSON * row = NULL;
+    char* response = NULL;
+    char* header = "due ";
+    int header_size = strlen(header);
+    os_calloc(OS_MAXSTR, sizeof(char), response);
+    memcpy(response, header, header_size);
+    char* payload = response+header_size;
+    int payload_size = OS_MAXSTR-header_size;
+
+    while ((row = wdb_exec_row_stmt(stmt, &status))) {
+        bool row_fits = cJSON_PrintPreallocated(row, payload, payload_size, FALSE);
```

Comment:
I didn't know about this function. Nice!

Reply:
+ 1